src/lib/services/simulator.ts

```diff
@@ -109,6 +114,41 @@ export class SimulatorService implements ISimulatorService {
     return requirementsInstalled;
   }
 
+  public async checkVersionRequirements(): Promise<Record<string, string>> {
+    const missingVersions = {
+      docker: VERSION_REQUIREMENTS.docker,
+      node: VERSION_REQUIREMENTS.node,
+    };
+
+    try {
+      await this.checkVersion(VERSION_REQUIREMENTS.node, "node");
+      missingVersions.node = "";
```

Comment:
@denishacquin I would add the missing version in case the checkVersion fails instead of adding it to be removed later

Reply:
Got it, I did it like so to follow the same reasonning as for installs, but I agree that it makes less sense here. Done.